src/main/java/com/taskadapter/redmineapi/internal/RedmineDateUtils.java

```diff
@@ -1,9 +1,14 @@
 package com.taskadapter.redmineapi.internal;
 
+import java.text.ParseException;
+import java.util.Date;
+import javax.xml.bind.DatatypeConverter;
```

Comment:
XML and JSON are both supported. So while JSON is underspecified and does not offer a default datetime/date mapping XML (xml schema) does. So instead of playing catchup with all variants that are possible, I will use the build in tools. 

This is not some xml parsing class, but the class definied by JAXB - see:

http://download.oracle.com/otndocs/jcp/jaxb-2.2-mrel2a-oth-JSpec/    (Chapter 7.9.2)

jaxb is part of every current Oracle JDK, for IBM there is an indication that it is also true (the support jax-ws, which is just nothing without jaxb). The namespace is not private, so I see no indication, that this will go away in the middle term.